internal/commands/accesslist/update.go

```diff
@@ -0,0 +1,151 @@
+package accesslist
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/10gen/realm-cli/internal/utils/flags"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type updateInputs struct {
+	cli.ProjectInputs
+	Address    string
+	NewAddress string
+	Comment    string
+}
+
+// CommandMetaUpdate is the command meta for the `accesslist update` command
+var CommandMetaUpdate = cli.CommandMeta{
+	Use:         "update",
+	Display:     "accesslist update",
+	Description: "Modify an IP address or CIDR block in the Access List of your Realm app",
+	HelpText: `Changes an existing entry from the Access List of your Realm app. You will be
+prompted to select an IP address or CIDR block if none is provided in the initial command.`,
+}
+
+// CommandUpdate is the `accesslist update` command
+type CommandUpdate struct {
+	inputs updateInputs
+}
+
+// Flags is the command flags
+func (cmd *CommandUpdate) Flags() []flags.Flag {
+	return []flags.Flag{
+		cli.AppFlagWithContext(&cmd.inputs.App, "to modify an entry in its Access List"),
+		cli.ProjectFlag(&cmd.inputs.Project),
+		cli.ProductFlag(&cmd.inputs.Products),
+		flags.StringFlag{
+			Value: &cmd.inputs.Address,
+			Meta: flags.Meta{
+				Name: "ip",
+				Usage: flags.Usage{
+					Description: "Specify the existing IP address or CIDR block that you would like to modify",
+				},
+			},
+		},
+		flags.StringFlag{
+			Value: &cmd.inputs.NewAddress,
+			Meta: flags.Meta{
+				Name: "new-ip",
+				Usage: flags.Usage{
+					Description: "Specify the new IP address or CIDR block that will replace the existing entry",
```

Comment:
seems like this flag too is "optional" (in the same sense that `--comment` is).  if that is the case, i would opt to just remove that `Note` from each flag and leave the usages a little simpler/shorter